index.html

```diff
@@ -14,14 +14,35 @@
       rel="stylesheet"
     />
 
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.realdevsquad.com/index.html" />
     <meta property="og:title" content="Real Dev Squad" />
     <meta
       property="og:description"
       content="Community of developers who collaborate to build Real software."
     />
     <meta property="og:image" content="img/Real-Dev-Squad@2x.png" />
-    <meta property="og:url" content="https://www.realdevsquad.com/index.html" />
-    <meta name="twitter:card" content="summary_large_image" />
+
+    <!-- Twitter Meta Tag-->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta
+      property="twitter:url"
+      content="https://www.realdevsquad.com/index.html"
+    />
+    <meta property="twitter:title" content="Real Dev Squad" />
+    <meta
+      property="twitter:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta property="twitter:image" content="img/Real-Dev-Squad@2x.png" />
```

Comment:
Can we please use the image provided in the description ??

Reply:
Sure! i will do the change, actually I did that because it was there in the previous meta tags.